frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -16,24 +17,52 @@ class MainSearch extends Component {
     this.setState({ searchTerm: event.target.value });
   }
 
+  handleValidation() {
+    let error = '';
+    let isValid = true;
+    const { searchTerm } = this.state;
+    if (!searchTerm) {
```

Comment:
I'd probably condense these two lines into `if (!this.state.searchTerm) {`, since we're not using `searchTerm` anywhere else.

Reply:
The linter is currently set up to require destructuring state and props.

I'm not super tied to this particular rule, but I do think it aids consistency & readability across the codebase. It also helps with refactoring, since if we start using `searchTerm` elsewhere, or turn `MainSearch` into a stateless component at some point, we only have to deal with one occurrence of `this.state`.

Reply:
Ahh, that's fair. I don't really see any reason to change the rule, it just wasn't one I was familiar with.